projects/azex/index.js

```diff
@@ -0,0 +1,21 @@
+const { sumTokensExport } = require("../helper/unwrapLPs")
+
+const ADDRESSES = require("../helper/coreAssets.json")
+const { BERA_CONTRACTS, UNICHAIN_CONTRACTS } = require("./infos")
+
+const config = {
+  berachain: {
+    tokens: Object.values(ADDRESSES.berachain),
+    owners: Object.values(BERA_CONTRACTS.addresses),
+  },
+  unichain: {
+    tokens: Object.values(ADDRESSES.unichain),
+    owners: Object.values(UNICHAIN_CONTRACTS.addresses),
```

Comment:
would restrict this to the list of addresses where users deposit funds rather than check all protocol related addresses? also, is there way to programatically get list of supported tokens, maybe via event log when a new orderbook pair is set up?